lib/gossip/state_transitions.js

```diff
@@ -60,33 +70,51 @@ Suspicion.prototype.start = function start(member) {
         return;
     }
 
-    if (this.timers[member.address]) {
+    var timer = this.timers[member.address];
+    if (timer) {
+        if (timer.state === state) {
+            this.logger.warn('redundant call to schedule a state transition for member, ignored', {
+                local: this.ringpop.whoami(),
+                suspect: member.address,
```

Comment:
In here everything was called `member`, and I left it as it was. Do you think we should be consistent in both codebases and rename `member` to `subject`?